xdem/spatial_tools.py

```diff
@@ -398,6 +408,149 @@ def hillshade(dem: Union[np.ndarray, np.ma.masked_array], resolution: Union[floa
     # The output is scaled by "(x + 0.6) / 1.84" to make it more similar to GDAL.
     return np.clip(255 * (shaded + 0.6) / 1.84, 0, 255).astype("float32")
 
+def get_xy_rotated(raster: gu.georaster.Raster, myang: float) -> tuple[np.ndarray, np.ndarray]:
+    """
+    Rotate x, y axes of image to get along- and cross-track distances.
+    :param raster: raster to get x,y positions from.
+    :param myang: angle by which to rotate axes (degrees)
+    :returns xxr, yyr: arrays corresponding to along (x) and cross (y) track distances.
+    """
+    # creates matrices for along and across track distances from a reference dem and a raster angle map (in radians)
+
+    myang = np.deg2rad(myang)
+
+    # get grid coordinates
+    xx, yy = raster.coords(grid=True)
+    xx = xx - np.min(xx)
```

Comment:
`xx -= xx.min()`, or does this do something more specific?

Reply:
Remnants from a previous version of the code